src/lib/cmdInterpreterAPI.js

```diff
@@ -40,6 +41,9 @@ class CmdInterpreterAPI {
       'remix.debugHelp()': 'Display help message for debugging'
     }
   }
+  async call (message) {
+    return await this._components.terminal.externalApi.request(message)
```

Comment:
if `this._components.terminal.externalApi.request(message)` is a Promise, you don't need `async/await` here. It's redoundant.